src/psyclone/tests/test_files/gocean1p0/model_mod.f90

```diff
@@ -30,12 +29,14 @@
 ! OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 ! OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ! -----------------------------------------------------------------------------
-! Author J.Henrichs, Bureau of Meteorology
+! Author: J.Henrichs, Bureau of Meteorology
```

Comment:
Please update the copyright date on this file.

Reply:
Done